### DISCUSSION TITLE
Add Cisco Badge

### DISCUSSION THREAD

**Discussion by Leandromeda** [Ileriayo/markdown-badges#409] [Ideas]
Please! lol

**Comment by tyler-holm** [comment]
Here you go:
![Cisco](https://img.shields.io/badge/cisco-%23049fd9.svg?style=for-the-badge&logo=cisco&logoColor=%23000000)

```
![Cisco](https://img.shields.io/badge/cisco-%23049fd9.svg?style=for-the-badge&logo=cisco&logoColor=%23000000)
```

I've also made a PR to merge this into master
